Casks/neovide.rb

```diff
@@ -0,0 +1,14 @@
+cask "neovide" do
+  version "0.8.0"
+  sha256 "73ee776483161dad707bfaa118b79a9946f10223624e67127e6ecf73fb6eb626"
+
+  url "https://github.com/neovide/neovide/releases/download/#{version}/Neovide.dmg.zip"
+  name "neovide"
+  desc "No Nonsense Neovim Client in Rust"
```

Comment:
```suggestion
  desc "Neovim Client"
```
"Do not use empty marketing jargon"
https://docs.brew.sh/Cask-Cookbook#stanza-desc